src/app-router.js

```diff
@@ -28,7 +28,11 @@ export class AppRouter extends Router {
   reset() {
     super.reset();
     this.maxInstructionCount = 10;
-    this._queue = [];
+    if (!this._queue) {
+      this._queue = [];
```

Comment:
If the property is never reassigned we may as well initialize it inline and skip the check here.